webrtc_test.go

```diff
@@ -61,12 +61,12 @@ func TestOnSignalingStateChangeCallback(t *testing.T) {
 }
 
 func TestOnIceCandidateCallback(t *testing.T) {
-	success := make(chan string, 1)
-	pcA.OnIceCandidate = func(c string) {
-		success <- c
+	t.SkipNow() // Can't import C in tests
```

Comment:
Yup, the lack of cgo in tests is unfortunate :/
In `data/`, I got around this using Go helpers, which wrap a cgo helper. Kinda cludgy but does allow testing the full callback path.
(haven't gotten around to updating the other PeerConnection callback tests in the root package in this way, but we can do that later)